executor-service/src/main/java/executor/service/service/step/impl/StepExecutionClickCss.java

```diff
@@ -14,7 +14,7 @@ public String getStepAction() {
 
     @Override
     public void step(WebDriver webDriver, Step step) {
-        String cssSelector = step.getAction();
+        String cssSelector = step.getValue();
         WebElement webElement = webDriver.findElement(By.cssSelector(cssSelector));
```

Comment:
You can click on an element without creating a new one:
```webDriver.findElement(By.cssSelector(cssSelector)).click();```

Reply:
I agree)